modules.d/95nvmf/parse-nvmf-boot-connections.sh

```diff
@@ -87,7 +97,17 @@ done
 [ -f "/etc/nvme/hostid" ] || exit 0
 
 if [ -f "/etc/nvme/discovery.conf" ] ; then
-    /sbin/initqueue --onetime --unique --name nvme-discover /usr/sbin/nvme connect-all
+    if [ "$trtype" = "tcp" ] ; then
+        /sbin/initqueue --settled --onetime --unique --name nvme-discover /usr/sbin/nvme connect-all
+        echo "exit 0" > /tmp/net.nvmf.did-setup # hack to fool rd.neednet=1, FIXME
```

Comment:
Please get in touch with me about that. We should fix that properly. //cc @tblume

Reply:
I spoke to Hannes yesterday, he already instructed me on how this could be fixed. But I'll ping you for sure.